README.md

```diff
@@ -92,6 +93,24 @@ The above output includes only software that is visible in the container (i.e.,
 syft <image> --scope all-layers
 ```
 
+#### Format conversion - __experimental__
+
+Syft can convert between different SBOM formats, such as CycloneDX, SPDX, and Syft's own format. The ability to convert SBOMs helps saving the time and effort it would take to recreate SBOMs from container images, or file systems.
+
+``` bash
+syft convert sbom.syft.json -o cyclonedx-json=sbom.cdx.json
```

Comment:
I would generalize this introduction of the command, like you'd see in a "synopsis" view of a CLI command — instead of starting with a specific example. Maybe something like:

```
syft convert <ORIGINAL-SBOM-FILE> -o <NEW-SBOM-FORMAT>[=<NEW-SBOM-FILE>]
```

And then maybe include an example at the end of this "format conversion" section